magik-debug-adapter/src/main/java/nl/ramsolutions/sw/magik/debugadapter/Main.java

```diff
@@ -2,13 +2,14 @@
 
 import java.io.IOException;
 import java.io.InputStream;
+import java.net.InetSocketAddress;
+import java.net.SocketAddress;
+import java.nio.channels.AsynchronousServerSocketChannel;
+import java.nio.channels.AsynchronousSocketChannel;
+import java.nio.channels.Channels;
+import java.util.concurrent.ExecutionException;
 import java.util.logging.LogManager;
-import org.apache.commons.cli.CommandLine;
-import org.apache.commons.cli.CommandLineParser;
-import org.apache.commons.cli.DefaultParser;
-import org.apache.commons.cli.Option;
-import org.apache.commons.cli.Options;
-import org.apache.commons.cli.ParseException;
+import org.apache.commons.cli.*;
```

Comment:
Please don't do this. This hides the used imports and is perhaps a sign of too much "complexity" in one file.

Reply:
I think this happens when I commit in my IDE, I didn't pay it much mind since it's just imports 😅. I can refactor my PRs if you want?